docs/source/blocks/block-style-wrapper.md

```diff
@@ -344,3 +344,28 @@ Each block in the Block Engine has a main wrapper with an automatic class name `
 ```
 
 You can use it for further control over the positioning and layout of the block.
+
+## Custom style wrapper build style object enhancer
+
+The style wrapper has a helper method that generates a style object from the block data.
```

Comment:
```suggestion
The style object builder has a helper method that generates a style object from the block data.
```

Reply:
This one is fine I think, I'd leave it as it is.